app/resources/api/v2/tube_rack_resource.rb

```diff
@@ -29,17 +29,25 @@ class TubeRackResource < BaseResource
       # Associations:
       has_many :racked_tubes
       has_many :comments, readonly: true
-      has_one :purpose, foreign_key: :plate_purpose_id
+      # TODO: change to purpose_id
+      has_one :purpose, foreign_key: :plate_purpose_id, class_name: 'TubeRackPurpose'
+      has_many :parents, readonly: true, polymorphic: true
+      has_many :state_changes, readonly: true
+      has_one :custom_metadatum_collection, foreign_key_on: :related
+      has_many :ancestors, readonly: true, polymorphic: true
+      # NB. no child or descendent associations as tube racks can't have children (tubes have children).
```

Comment:
I don't think it's true that tube racks can't have children?

Reply:
we discussed this and decided it was the tubes that have children:
1. parent plate has child tube rack
2. tube rack has racked tubes (not children)
3. tubes have child plate

what would the child of a tube rack be?

Reply:
I think in the specific scRNA Core case that's true, but not necessarily in the general case. 

If it was just a simple plate --> tube rack --> plate, then I could see all the parent-child relationships being from the tube rack.